app/controllers/team_members_controller.rb

```diff
@@ -1,6 +1,8 @@
 class TeamMembersController < ApplicationController
 
   def index
-    @team_members = TeamMember.order(name: :asc)
+    @team_members = TeamMember.where(is_leader: false).order(name: :asc)
+    @leaders = TeamMember.where(is_leader: true).order(name: :asc)
```

Comment:
Rather than 2 queries here, I suggest you just use `select`.

```
@team_members = TeamMember.order(name: :asc)
@leaders = @team_members.select { is_leader: true }
```

Then in the view, when rendering team members, `@team_members.select { is_leader: false }.each do ....`

This way you only make 1 query to the DB, and then you're just operating on a collect with Ruby, which is much faster. Ultimately, not a huge difference here in terms of final performance because there will never be >100,000 team members; but, it's a good habit to get into to reduce DB load where possible.